rel/server/env.sh.eex

```diff
@@ -1,4 +1,4 @@
-export RELEASE_NODE=livebook_server
+[ -z "${RELEASE_NODE}" ] && export RELEASE_NODE=livebook_server
```

Comment:
```suggestion
export RELEASE_NODE="${RELEASE_NODE:-livebook_server}"
```

Reply:
yeah... i thought about that...

Reply:
but also... i thought about "this can be run in sh not in the bash"

Reply:
Hrm, Elixir release files use this style anyway, so we should be fine. It also passes shellcheck. :)

Reply:
awesome 😉